lib/broccoli/angular-broccoli-less.js

```diff
@@ -42,13 +42,13 @@ class LESSPlugin extends Plugin {
   }
 }
 
-exports.makeBroccoliTree = (sourceDir) => {
+exports.makeBroccoliTree = (sourceDir, options) => {
   if (less) {
     let lessSrcTree = new Funnel(sourceDir, {
       include: ['**/*.less'],
       allowEmpty: true
     });
 
-    return new LESSPlugin([lessSrcTree]);
+    return new LESSPlugin([lessSrcTree], options);
   }
 };
```

Comment:
You could also add additional options same as you did for `sass`. And then call it; 
`less.render(content, this.options)`.